src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const cleanStrings = sourceString
+    .trim()
+    .split(';')
+    .filter((style) => style.length);
+
+  cleanStrings.forEach((cleanString) => {
+    const partsOfString = cleanString
+      .split(':')
+      .map((partStyle) => partStyle.trim());
+
+    const key = partsOfString[0];
+    const value = partsOfString[1];
+
```

Comment:
do u need `map` in this case?
```suggestion

    const key = partsOfString[0].trim();
    const value = partsOfString[1].trim();

```